lib/events/dynamoevents/dynamoevents.go

```diff
@@ -709,7 +711,17 @@ func (l *Log) SearchEvents(fromUTC, toUTC time.Time, namespace string, eventType
 		eventArr = append(eventArr, event)
 	}
 
-	sort.Sort(byTimeAndIndex(eventArr))
+	var toSort sort.Interface
+	switch order {
+	case types.EventOrderAscending:
+		toSort = byTimeAndIndex(eventArr)
+	case types.EventOrderDescending:
+		toSort = sort.Reverse(byTimeAndIndex(eventArr))
+	default:
+		return nil, "", trace.BadParameter("invalid event order")
```

Comment:
If you decide to switch to a proto enum for ordering, take advantage of its generated string method to improve the error messages such as this.  Ex:

```suggestion
		return nil, "", trace.BadParameter("invalid event order: %s", order)
```

Reply:
Done